### DISCUSSION TITLE
Removing title, description, and updated/created as required frontmatter

### DISCUSSION THREAD

**Discussion by dionysuzx** [dendronhq/dendron#2912] [RFCs/Ideas]
I have a feature request for reducing the required frontmatter to only the id. And if this is the case then the title would be automatically inferred from the hyphenated note name. Is this possible?

**Comment by dionysuzx** [comment]
i understand finding the balance between recommending/enforcing some features in the metadata, but i would like to encourage, not enforce the usage of dendron on my team. meaning it would be nice to have a linter so that they are following a specification, not requiring them to use the dendron tooling. i do however think everyone will use the tooling... i just don't like to say it is required but rather a nice to have in our markdown repo. if an id is all that dendron really needs for our publish, then i would like to restrict the number of fields in the frontmatter to only be an id where if the dev really wanted to, could randomly generate their own and have the linter / doctor check for an id collision.

**Comment by aleksey-rowan** [comment]
Backlinking to the original issue:

- https://github.com/dendronhq/dendron/issues/2971

And another issue about potentially removing timestamps from the frontmatter:

- https://github.com/dendronhq/dendron/issues/1855
